x-pack/filebeat/module/rabbitmq/log/_meta/fields.yml

```diff
@@ -0,0 +1,17 @@
+- name: log
+  type: group
+  description: >
+    RabbitMQ log files
+  fields:
+    - name: pid
+      type: keyword
+      description: The Erlang process id
+      example: <0.222.0>
+    - name: level
+      type: alias
+      path: log.level
+      migration: true
```

Comment:
This migration aliases are only used for modules which were introduced before 7.0. For the new modules like this one we don't need any aliases. Having only `log.level` is enough and this entry can be removed.